### ISSUE TITLE
createCipheriv support

### ISSUE THREAD

**Issue by tamashalasi** [tradle/react-native-crypto#25]
There is no createCipheriv method in this library while [there is in the Node.js one.](https://nodejs.org/api/crypto.html#crypto_crypto_createcipheriv_algorithm_key_iv_options).
Firstly, is the support planned?
Secondly, I have a special use case, where the `iv` is an empty string. Can I accomplish that somehow with the existing methods?

**Comment by tamashalasi** [comment]
createDecipheriv is missing too.

**Comment by mvayngrib** [comment]
@trustedtomato this module is a fork of [crypto-browserify](https://github.com/crypto-browserify/crypto-browserify). At the moment it doesn't implement anything additional. If you find a reliable shim for createCipheriv that works in React Native, please link it here :)

**Comment by BrodaNoel** [comment]
@mvayngrib I was thinking that `crypto-browserity` support it.
Check this: https://github.com/crypto-browserify/crypto-browserify/blob/master/index.js#L23

**Comment by mvayngrib** [comment]
@BrodaNoel you're right, it has been a while since I checked upstream. Want to give updating this module a try?

**Comment by sayem314** [comment]
Any updates? Especially looking for decipheriv!

**Comment by samaneh-kamalian** [comment]
why createCipheriv does not accept key that was created in pbkdf2?

**Comment by javiesses** [comment]
Hi team! Any updates on this? Is there any plan to support them?
